website/src/pages/api/update_task.ts

```diff
@@ -18,6 +19,8 @@ const handler = async (req, res) => {
     return;
   }
 
+  const oasstApiClient = new OasstApiClient(process.env.FASTAPI_URL, process.env.FASTAPI_KEY);
```

Comment:
Instead of creating a new client object every render, can we create a single client object in `oasst_api_client.ts` and export that?  This is similar to how we do things with the prisma client.

Reply:
We could do this, but if we wanted to add the 'user' to the constructor (as mentioned in https://github.com/LAION-AI/Open-Assistant/issues/353#issuecomment-1370478078) it would need to be reworked. 

Is there some sort of request context that we could add the api client to?

Reply:
We don't need to provide `user` in the constructor of the client.  Keeping it as an argument to the method calls seems fine.  The cleanup we need is just for the usage of `user` to check what type of user is it and send a different user ID if they're authenticated via discord.

NextJS Middleware seems too intrusive since we don't need this to happen for every single API route.

Reply:
Might be fixed by 3c330dc
Added prisma-like global variable init & changed exports so both `oasstApiClient` and `OasstApiClient` class could be used.